sechub-pds/src/main/java/com/daimler/sechub/pds/PDSSystemPropertyInjector.java

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.pds;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * This component injects some special PDS Spring Boot values into corresponding
+ * system properties. So we can configure those parts in our `application.yaml`
```

Comment:
Those properties are specific to the JVM not to the operating system.

```suggestion
 * JVM system properties. So we can configure those parts in our `application.yaml`
```